ibaqpy/ibaq/imputation_methods.py

```
@@ -0,0 +1,102 @@
+import numpy as np
+import pandas as pd
+
+from ibaqpy.ibaq.ibaqpy_commons import PEPTIDE_CANONICAL, PROTEIN_NAME, CONDITION, NORM_INTENSITY
+
+
+
+
+def impute_missing_values(
+    data: Optional[Union[pd.DataFrame, List[pd.DataFrame]]],
```

Ruff (0.8.0)

Undefined name `Optional`

(F821)